Added proxy specific properties

JaniruTEC opened the pull request.

JaniruTEC committed:
Added proxy specific properties

SailReal commented on Advanced-Settings.md:
```diff
@@ -12,37 +12,44 @@ Cryptomator allows configuring parts of the environment using *properties.* For
+| java.net.useSystemProxies=[Boolean]                         | Whether Cryptomator should use the system proxy settings (true) or no proxy (false) when connecting to the internet (e.g. for update checks).                      | All   |
+| https.proxyHost=[Host]                                      | The host of the proxy server to use for connecting to the internet (e.g. for update checks). Overrides `java.net.useSystemProxies=[Boolean]`.                      | All   |
+| https.proxyPort=[Port]                                      | The port of the proxy server to use for connecting to the internet (e.g. for update checks). Requires `https.proxyHost=[Host]` to be set.                          | All   |
+| http.nonProxyHosts=[NonProxyHostList]                       | The hosts that are not accessed through the proxy server when connecting to the internet (e.g. for update checks). Defaults to `localhost\|127.*\|[::1]` if unset. | All   |
```
Would remove the word internet completely.

```suggestion
| https.proxyHost=[Host]                                      | The host of the proxy server to use for connecting to the remote (e.g. for update checks). Overrides `java.net.useSystemProxies=[Boolean]`.                      | All   |
| https.proxyPort=[Port]                                      | The port of the proxy server to use for connecting to the remote (e.g. for update checks). Requires `https.proxyHost=[Host]` to be set.                          | All   |
| http.nonProxyHosts=[NonProxyHostList]                       | The hosts that are not accessed through the proxy server when connecting to the remote (e.g. for update checks). Defaults to `localhost\|127.*\|[::1]` if unset. | All   |
```

JaniruTEC replied:
Resolved by b9dee9d34fd743707e3c5a9f6c0806a167e86446.

SailReal reviewed (changes requested):
Thank you, just one minor change.

JaniruTEC committed:
Added extra spacing to make future edits less invasive

Also reformatted document

JaniruTEC committed:
Adjusted wording

Resolves: https://github.com/JaniruTEC/cryptomator-wiki/pull/1#discussion_r1588941519
